### PR TITLE
Cherry-pick: Fix for vic machine server publish

### PR DESCRIPTION
**Pull request by mdharamadas1**
cherry-pick https://github.com/vmware/vic/pull/6902

### CODE REVIEW

File: tests/resources/VCH-Util.robot [hunk 1]
```diff
@@ -371,9 +371,10 @@ Gather Logs From Test Server
     ${out}=  Run  curl -k -b vic-admin-cookies %{VIC-ADMIN}/container-logs.zip -o ${SUITE NAME}-%{VCH-NAME}-container-logs${name-suffix}.zip
     Log  ${out}
     Remove File  vic-admin-cookies
-    ${out}=  Run  govc datastore.download %{VCH-NAME}/vmware.log %{VCH-NAME}-vmware${name-suffix}.log
+    ${host}=  Get VM Host Name  %{VCH-NAME}
```
Comment (anchal-agrawal) [inline tests/resources/VCH-Util.robot]: These changes aren't part of https://github.com/vmware/vic/pull/6902 - where are they from?

Comment (mdharamadas1) [reply tests/resources/VCH-Util.robot]: i have no idea why these are getting included, looking now

Comment (anchal-agrawal) [reply tests/resources/VCH-Util.robot]: It's a good idea to cherry-pick the commit from the other branch directly. I usually first merge to master, grab the commit from the master branch, apply it to the releases branch, then create the cherry-pick PR.

Comment (mdharamadas1) [reply tests/resources/VCH-Util.robot]: ok, fixed!

**Comment by mdharamadas1** [comment]
CI build in progress.

**Pull request merged**